src/js/other-websites/inpage_menu.js

```diff
@@ -49,13 +47,13 @@ async function handleKeydownEvents (e) {
 
 async function isUserSignedIn() {
   const userApiToken = await browser.storage.local.get("apiToken");
-  return userApiToken.hasOwnProperty("apiToken");
+  return Object.prototype.hasOwnProperty.call(userApiToken, "apiToken");
 }
 
-async function getServerStoragePref() {
+async function getCachedServerStoragePref() {
```

Comment:
@maxxcrawford I renamed this, because this was shadowing a function that's also defined in `background.js` (and hence listed as global in the `.eslintrc`), but that did something different.